diff -ruN /home/nigel/websites/PHPMailer_v5.1/class.phpmailer.php phpmailer/class.phpmailer.php
--- /home/nigel/websites/PHPMailer_v5.1/class.phpmailer.php	2009-11-11 11:04:14.000000000 +1100
+++ phpmailer/class.phpmailer.php	2010-11-30 14:11:54.536232002 +1100
@@ -306,13 +306,13 @@
   // PROPERTIES, PRIVATE AND PROTECTED
   /////////////////////////////////////////////////
 
-  private   $smtp           = NULL;
+  protected   $smtp           = NULL;
   private   $to             = array();
   private   $cc             = array();
   private   $bcc            = array();
   private   $ReplyTo        = array();
   private   $all_recipients = array();
-  private   $attachment     = array();
+  protected   $attachment     = array();
   private   $CustomHeader   = array();
   private   $message_type   = '';
   private   $boundary       = array();
@@ -321,7 +321,7 @@
   private   $sign_cert_file = "";
   private   $sign_key_file  = "";
   private   $sign_key_pass  = "";
-  private   $exceptions     = false;
+  protected   $exceptions     = false;
 
   /////////////////////////////////////////////////
   // CONSTANTS
@@ -1065,6 +1065,9 @@
     }
   }
 
+  public function GetMessageDate() {
+    return self::RFCDate();
+  }
   /**
    * Assembles message header.
    * @access public
@@ -1078,7 +1081,7 @@
     $this->boundary[1] = 'b1_' . $uniq_id;
     $this->boundary[2] = 'b2_' . $uniq_id;
 
-    $result .= $this->HeaderLine('Date', self::RFCDate());
+    $result .= $this->HeaderLine('Date', $this->GetMessageDate());
     if($this->Sender == '') {
       $result .= $this->HeaderLine('Return-Path', trim($this->From));
     } else {
@@ -1293,7 +1296,7 @@
    * @access private
    * @return void
    */
-  private function SetMessageType() {
+  protected function SetMessageType() {
     if(count($this->attachment) < 1 && strlen($this->AltBody) < 1) {
       $this->message_type = 'plain';
     } else {
